pkg/storage/cache/cache.go

```diff
@@ -1,8 +1,37 @@
 package cache
 
-import "time"
+import (
+	"time"
 
-type Cache interface {
```

Comment:
Do not remove this interface. Keep the interface as it is, and create a new package which implements the interface. This is what we do here: https://github.com/scratchdata/scratchdata/tree/main/pkg/storage/queue

Reply:
Can you please elaborate more on the structure of storage/cache directory ?

Reply:
The `storage/cache` directory might have many implementations of a cache (for example, in-memory and redis.) We use a go interface in order to specify how each cache should behave.

Create a new folder called `memory` under `storage/cache` where you have your implementation with your `Cache` struct. Then, any other place in the code which uses a cache can pass the interface type, regardless of the underlying implementation.

You may look at the `storage/queue` directory structure for a template on how to do this.

Reply:
@poundifdef I have created the directories. Can you please tell how to call new cache now ?

<img width="565" alt="image" src="https://github.com/scratchdata/scratchdata/assets/64700961/93117070-053f-438d-95c2-46aa60d90fce">

Reply:
Thanks! This is resolved.